elements.js

```diff
@@ -0,0 +1,4 @@
+export const taskForm = document.getElementById("taskForm");
+export const taskTitle = document.getElementById("taskTitle");
+export const taskList = document.getElementById("taskList");
+export const btnCreate = document.querySelector('.btn-create');
```

Comment:
```suggestion
export const $taskForm = document.getElementById("taskForm");
export const $taskTitle = document.getElementById("taskTitle");
export const $taskList = document.getElementById("taskList");
export const $btnCreate = document.querySelector('.btn-create');
```

It is the code convention to add `$` sign while declaring DOM elements to keep this separate from the other variables.